homeassistant/components/enigma2/strings.json

```diff
@@ -11,6 +11,14 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        },
+        "data_description": {
+          "host": "The hostname or address of the device.",
+          "port": "The port number of the OpenWebif instance (default 80 for HTTP and 443 for HTTPS).",
+          "ssl": "Use HTTPS to connect to OpenWebIf.",
```

Comment:
Also, how does the user see this URL? If you have to access the webpage to actually do something with the device, we can consider using `url` as parameter, and then just split it up into pieces in the config flow. This way the user can just copy and paste, rather than split everything. Another example for this is `overseerr`